composer.json

```
@@ -12,13 +12,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "ext-oci8": ">=2.0.0",
-        "doctrine/dbal": ">=2.3,<2.8-dev"
+        "php": "^7.1",
+        "ext-oci8": ">=2.1.2",
```

You don't have to specify any version for PHP extensions.

```suggestion
        "ext-oci8": "*",
```